source/fundamentals/builders/sort.txt

```diff
@@ -11,3 +11,326 @@ Sorts Builders
+
+   {"_id": 1, "letter": "c", "food": "coffee with milk"}
+   {"_id": 3, "letter": "a", "food": "maple syrup"}
+   {"_id": 4, "letter": "b", "food": "coffee with sugar"}
+   {"_id": 5, "letter": "a", "food": "milk and cookies"}
+   {"_id": 2, "letter": "a", "food": "donuts and coffee"}
+   {"_id": 6, "letter": "c", "food": "maple donut"}
+
+
+Sorting Direction
+-----------------
+
+The ``Sorts`` class provides methods for specifying the direction of your sort.
+The direction of your sort can either be **ascending**, or **descending**.
+An ascending sort orders your data from smallest to largest. A
+descending sort orders your data from largest to smallest.
+
+Here are some examples of data sorted in ascending order:
+
+* Numbers: 1, 2, 3, 43, 43, 55, 120
+* Dates: 1990-03-10, 1995-01-01, 2005-10-30, 2005-12-21 
+* Words (Alphabetical): carrot, cheese, cucumber, hummus
+
+Here are some examples of data sorted in descending order:
+
+* Numbers: 100, 30, 12, 12, 9, 3, 1
+* Dates: 2020-01-01, 1998-12-11, 1998-12-10, 1900-07-22 
+* Words (Alphabetical): pear, mango, grapes, apple
+
+The following subsections show how to specify these sorts using
+the ``Sorts`` class.
+
+Ascending
+~~~~~~~~~
+
+To specify an ascending sort, use the ``Sorts.ascending()`` static
+factory method. Pass ``Sorts.ascending()``
+the name of the field you need to sort on.
+
+The ``ascending()`` method can be used as follows:
+
+.. code-block:: java
+
+   import static com.mongodb.client.model.Sorts.ascending;
+
+   // <MongoCollection setup code here>
+
+   collection.find().sort(ascending("<field name>"));
+
+The above ``sort()`` method returns a 
+:java-docs:`FindIterable </apidocs/mongodb-driver-sync/com/mongodb/client/FindIterable.html>`
+object containing the documents in your collection, sorted from smallest
+to largest on the specified field name. 
+
+.. example::
+
+   In this code example, we use the ``ascending()`` method to sort the
+   :ref:`sort_example collection <sorts-builders-sort-example>`  
+   by the ``_id`` field:
+
+   .. code-block:: java
+   
+      import static com.mongodb.client.model.Sorts.ascending;
+   
+      // <MongoCollection setup code here>
+      
+      Bson idSort = ascending("_id");
+      List<Document> results = new ArrayList<>();
+      collection.find().sort(idSort).into(results);
+      for (Document result : results) {
+          System.out.println(result.toJson());
+      }
+
+   The output of the code snippet above should look something like this: 
+   
+   .. code-block:: json
+   
+      {"_id": 1, "letter": "c", "food": "coffee with milk"}
+      {"_id": 2, "letter": "a", "food": "donuts and coffee"}
+      {"_id": 3, "letter": "a", "food": "maple syrup"}
+      ...
+
+Descending
+~~~~~~~~~~
+
+To specify a descending sort, use the ``Sorts.descending()`` static factory
+method. Pass ``Sorts.descending()`` the name of the field you need to sort on.
+
+The following code snippet shows how to specify a descending sort on the
```

Comment:
Yes, the reason is that the `Descending` code snippet contains significantly less information than the `Ascending` code example. This code snippet style is repeated in `Handling Ties` as well.

Reply:
Since we haven't established this pattern across the Java docs nor explained it to the user, would it make sense to avoid using the "example" directives for now and -- since it seems like it could be a useful signal -- create a ticket to revisit whether we should update this across all the documentation as well as provide sufficient explanation?